package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "fh-wfm-signature",
-  "version": "0.0.15",
+  "name": "wfm-signature",
```

Comment:
I think you can omit this file from your changes or it would change the name of the package